en/concepts/scene/light/sphere-light.md

```diff
@@ -1,14 +1,14 @@
-# Spherical Light
+# Spherical Lights
 
-Cocos Creator 3.0 uses Spherical Light instead of **Point Light**, because Point Light ignore volume, but all Physical Lights in the real world have the lights `size` property.
+Cocos Creator 3.0 uses Spherical Lights instead of the **Point Light**, because the point light ignore volume, but all Physical Lights in the real world have the lights `size` property.
```

Comment:
Cocos Creator 3.0 uses Spherical Light instead of **Point Light** because  Point Light ignores volume, but physical light in the real world have a light `size` property.